src/hypothesis/strategies.py

```diff
@@ -713,65 +714,69 @@ def characters(whitelist_categories=None, blacklist_categories=None,
     """Generates unicode text type (unicode on python 2, str on python 3)
     characters following specified filtering rules.
 
-    When no filtering rules are specifed, any character can be produced.
-
-    If ``min_codepoint`` or ``max_codepoint`` is specifed, then only
-    characters having a codepoint in that range will be produced.
-
-    If ``whitelist_categories`` is specified, then only characters from those
-    Unicode categories will be produced. This is a further restriction,
-    characters must also satisfy ``min_codepoint`` and ``max_codepoint``.
-
-    If ``blacklist_categories`` is specified, then any character from those
-    categories will not be produced. This is a further restriction,
-    characters that match both ``whitelist_categories`` and
-    ``blacklist_categories`` will not be produced.
-
-    If ``whitelist_characters`` is specified, then any additional characters
-    in that list will also be produced.
-
-    If ``blacklist_characters`` is specified, then any characters in that list
-    will be not be produced. Any overlap between ``whitelist_characters`` and
-    ``blacklist_characters`` will raise an exception.
-
-    Examples from this strategy shrink towards smaller codepoints.
+    - When no filtering rules are specifed, any character can be produced.
+    - If ``min_codepoint`` or ``max_codepoint`` is specifed, then only
+      characters having a codepoint in that range will be produced.
+    - If ``whitelist_categories`` is specified, then only characters from those
+      Unicode categories will be produced. This is a further restriction,
+      characters must also satisfy ``min_codepoint`` and ``max_codepoint``.
+    - If ``blacklist_categories`` is specified, then any character from those
+      categories will not be produced.  Any overlap between
+      ``whitelist_categories`` and ``blacklist_categories`` will raise an
+      exception, as each character can only belong to a single class.
+    - If ``whitelist_characters`` is specified, then any additional characters
+      in that list will also be produced.
+    - If ``blacklist_characters`` is specified, then any characters in
+      that list will be not be produced. Any overlap between \
+      ``whitelist_characters`` and ``blacklist_characters`` will raise an
+      exception.
+
+    The ``_codepoint`` arguments must be integers between zero and
+    :obj:`python:sys.max_unicode`.  The ``_characters`` arguments must be
+    collections of length-one unicode strings (e.g. longer unicode strings).
```

Comment:
I find the "e.g. longer unicode strings" kinda confusing even knowing what it means! (also it's technically wrong: The unicode string could be one character). Maybe "this includes unicode strings" instead?